webui/react/src/hooks/useSettings.ts

```diff
@@ -11,10 +11,10 @@ import { clone, hasObjectKeys, isBoolean, isEqual, isNumber,
 import { Storage } from 'shared/utils/storage';
 import handleError from 'utils/error';
 
+import usePrevious from '../shared/hooks/usePrevious';
 import { Primitive, RecordKey } from '../shared/types';
```

Comment:
ditto

Reply:
https://github.com/determined-ai/determined/pull/4581 let's land that and then I make sure this pr doesn't readd those